### ISSUE TITLE
Add `alertId` to the BigQuery SuperNNova table

### ISSUE THREAD

**Issue by troyraen** [mwvgroup/pittgoogle-user#3]
It will need to be added in two places:

 - the file containing the schema for the BigQuery table
 - the file that actually sends the classifications to the table

**Comment by wmwv** [comment]
To be specific, am I correct to understand that that should be done here:
https://github.com/mwvgroup/pittgoogle-user/blob/main/templates/bq_elasticc_SuperNNova_schema.json
and here
https://github.com/mwvgroup/pittgoogle-user/blob/main/classifier/main.py#L120

I ask in the context of setting up the `MicroLIA` classifier.

**Comment by wmwv** [comment]
Should `diaObjectId` be an integer in the BigQuery table schema?
```
    {
      "description": "object identifier or name",
      "mode": "REQUIRED",
      "name": "diaObjectId",
      "type": "STRING"
    },
```

**Comment by troyraen** [comment]
> To be specific, am I correct to understand that that should be done here:
> https://github.com/mwvgroup/pittgoogle-user/blob/main/templates/bq_elasticc_SuperNNova_schema.json
> and here
> https://github.com/mwvgroup/pittgoogle-user/blob/main/classifier/main.py#L120

Yes that's correct.

**Comment by troyraen** [comment]
> Should `diaObjectId` be an integer in the BigQuery table schema?
> 
> ```
>     {
>       "description": "object identifier or name",
>       "mode": "REQUIRED",
>       "name": "diaObjectId",
>       "type": "STRING"
>     },
> ```

Yes, looks like it should be an integer ([schema](https://github.com/LSSTDESC/elasticc/tree/main/alert_schema), for reference). Not sure why we have it as a string. It might be that the module is getting it from the message metadata (which are always strings), but even so we should convert back to an integer to match the original schema.

**Comment by troyraen** [comment]
Fixed by #21
